packages/components/ContextualMenu/src/ContextualMenu.types.ts

```diff
@@ -3,31 +3,31 @@ import { IViewProps } from '@fluentui-react-native/adapters';
 import { IRenderData } from '@uifabricshared/foundation-composable';
 
 export const contextualMenuName = 'ContextualMenu';
-
 export interface ContextualMenuContext {
-  /*
-   ** The currently selected ContextualMenu's key
+  /**
+   * The currently selected ContextualMenu's key
    */
   selectedKey: string | null;
 
-  /*
-   ** Updates the clicked menu item and calls the client’s onItemClick callback
+  /**
+   * Updates the clicked menu item and calls the client’s onItemClick callback
    */
   onItemClick?: (key: string) => void;
-  /*
-   ** Parent menu's onDismiss callback that is passed into submenu to call when submenu item is clicked
+
+  /**
+   * Parent menu's onDismiss callback that is passed into submenu to call when submenu item is clicked
    */
   onDismissMenu?: () => void;
-  /*
-   ** Checks if any child menus are open
+
+  /**
+   * Checks if any child menus are open
    */
   isSubmenuOpen?: boolean;
-  /*
-   ** ContextualMenuItems will call this submenu dismissal when they are hovered
+
+  /**
+   * ContextualMenuItems will call this submenu dismissal when they are hovered
    */
   dismissSubmenu?: () => void;
-
-  setSubmenuItemHovered?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Since we're shipping this component right now, I would rather have this set as deprecated than to remove it outright, so that we don't introduce breaking changes.

Reply:
Spoke offline, this isn't external so this should be ok.